docs/cql/ddl.rst

```diff
@@ -544,6 +544,11 @@ options of a table are described in the following sections.
 Compact tables
 ``````````````
 
+Support for compact tables has been deprecated.
+By default, newly created tables cannot be defined with the ``COMPACT STORAGE`` option.
```

Comment:
Idea for slightly different wording (you "create" a table, not "define" one):
"By default, new tables cannot be created with the ``COMPACT STORAGE`` option, but existing tables continue to work as before.
To allow creating new COMPACT STORAGE tables,  set the ..."

Reply:
fixed wording as per you suggestion in v3.